hadoop-hdds/rocks-native/pom.xml

```diff
@@ -393,7 +393,7 @@
                     <argument>-h</argument>
                     <argument>${project.build.directory}/native/javah</argument>
                     <argument>${project.basedir}/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedRawSSTFileReader.java</argument>
-                    <argument>${project.basedir}/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedRawSSTFileReaderIterator.java</argument>
+                    <argument>${project.basedir}/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedRawSSTFileIterator.java</argument>
```

Comment:
Oh my bad ! Missed this in my previous commit. @raju-balpande Thanks for pointing this out.